src/pages/Legal/PromoTermsOfUse.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+	<www-page>
+		<div class="row page-content">
+			<tertiary-menu class="show-for-large large-2 tertiary-menu-terms">
+				<ul>
+					<li>
+						<router-link to="/legal/privacy">
+							Privacy Policy
+						</router-link>
+					</li>
+					<li>
+						<router-link to="/legal/terms-of-use-february-2020">
+							Terms of Use
```

Comment:
Seems like we may need to link to the existing terms from here too...and maybe add the upcoming terms to the original page menu as well. Let's check with Brandon this.

Reply:
Checked with Brandon, his comments on Jira, updated with new commit.